### ISSUE TITLE
Support for custom biomes

### ISSUE THREAD

**Issue by URSFR** [lichenaut/PlantNerfer#3]
I tried to add custom biomes from Terra and RealisticSeasons but it didnt work, it only supports from bukkit list biomes?

**Comment by lichenaut** [comment]
You're correct in saying PlantNerfer only supports those biomes. I'll play around with a Terra soft dependency in my free time, unless you have circumstances that are pressing?

**Comment by URSFR** [comment]
Don't worry, take your time, thank you

**Comment by lichenaut** [comment]
It seems that, in order to add support for custom biomes, I would have to maintain a different version of the plugin for each version of Minecraft the plugin supports, as it requires using obfuscated 'Minecraft-itself' functions, seen here: https://www.spigotmc.org/threads/1-17-getting-custom-biomes-and-dimensions-by-namespace.513957/ 

This makes me think adding this feature is too much for me, at least until more people ask for this, which means I will keep this open as a space for that. Thank you for your patience.

**Comment by tollendoponens** [comment]
Hello, I'd like to second this suggestion. However, I'm interested in a Terralith soft-dependency.

**Comment by Parthodys** [comment]
Does PlantNerfer not work with RealisticSeasons biomes?

**Comment by lichenaut** [comment]
> Does PlantNerfer not work with RealisticSeasons biomes?

Correct, non-vanilla biomes just have the CUSTOM type.
